src/base/ddl_log.c

```diff
@@ -479,30 +516,6 @@ logddl_set_commit_mode (bool mode)
     }
 }
 
-void
-logddl_set_jsp_mode (bool mode)
-{
-  if (ddl_logging_enabled)
-    {
-      ddl_audit_handle.jsp_mode = mode;
-    }
-}
-
-bool
-logddl_get_jsp_mode ()
-{
-  return ddl_audit_handle.jsp_mode;
-}
-
-static void
-logddl_set_elapsed_time (long sec, long msec)
-{
-  if (ddl_logging_enabled)
-    {
-      snprintf (ddl_audit_handle.elapsed_time, 20, "elapsed time %ld.%03ld", sec, msec);
-    }
-}
-
```

Comment:
Because you deleted above two functions, member 'jsp_mode' is not used anymore.
What about remove 'jsp_mode' from `struct t_ddl_audit_handle`

Reply:
Good point. 
Thank you.